src/spacegame3/gamedata/systems/CelestialBody.java

```diff
@@ -51,8 +51,8 @@ public CelestialBody(String name, Map<String, List<String>> data, CelestialBodyS
     }
 
     @Override
-    public boolean isLandable(){
-        return !tabInfo.isEmpty();
+    public boolean isLandable(){ //boolean for whether the space is landable
+        return !tabInfo.isEmpty(); 
     }
 
     public List<TabRecord> getTabs() {
```

Comment:
you could try to add the javadoc for this method and the next as well as for the constructor